pkg/codegen/python/gen.go

```diff
@@ -301,15 +297,11 @@ func (mod *modContext) genInit(exports []string) string {
 					child = child[match[2]:match[3]]
 				}
 			}
-			if i > 0 {
-				fmt.Fprintf(w, ",\n")
+			if child != "config" {
+				fmt.Fprintf(w, "    %s,\n", PyName(child))
```

Comment:
@pgavlin, do you know why we skip config? Avoiding side-effects or something else?

Reply:
I don't know why we skip this, no. I believe that this pattern was present in the old `tfgen` code generator and I copied it here.

Reply:
I did some spelunking and the description in https://github.com/pulumi/pulumi-terraform/pull/356 explains why:

```python
for pkg in __all__:
    if pkg != 'config': # The "config" package is not importable without an engine
        importlib.import_module(f"{__name__}.{pkg}")
```

> The "config" package is not importable without an engine

We may actually be able to do this now given more recent changes to decouple the engine for testing/mocking, but I have not tried it. @komalali, do you want to see if it works without any other changes?

My main feedback is: It'd be nice if we could import `config` too so it also shows up in intellisense, but if it requires more refactoring to be able to work, we could continue to exclude it for now and come back to it later. Unless it's an easy enough change to make now.

Reply:
Ooh thanks for digging into that. I'll try that.

Reply:
This appears to "just work" without any issues 🤞 🎉 . Tested this in the pulumi-azure provider.